pkg/export/service.go

```diff
@@ -133,6 +130,18 @@ func (e *exporter) Logger() logger.Logger {
 	return e.logger
 }
 
+func (e *exporter) newRoute(r config.Route) routes.Route {
+	natsTopic := fmt.Sprintf("%s.%s", NatsSub, r.NatsTopic)
```

Comment:
It's normal to have topic after subtopic?

Reply:
that is of nats subject